source/documentation/getting_started/quick_start.md

```diff
@@ -1,10 +1,37 @@
 ## Quick start guide
 
-Registers in beta are reliable and are ready for use in products and services.
+Registers which are in beta are reliable and ready for use in products and services. Each register has an open, RESTful API which you can use to access the data in the register. 
 
-1. [Find the base URL for the register(s) you want to use](https://registers.cloudapps.digital/registers). For example, for the `local-authority-eng` register: `https://local-authority-eng.register.gov.uk/`.
-2. [Find the endpoints you want](#apireference). For example, `/record/BIR` in `https://local-authority-eng.register.gov.uk/record/BIR`.
-3. Add `.json`, `.yaml`, `.csv`, `.tsv`, or `.ttl` to the end of the request URL (before any query parameters) to choose the format of the response, or amend headers to `accept application/json` or equivalent.
-4. Parse register data and use it in your product or service.
+You can follow this guide to quickly get started with GOV.UK Registers APIs. 
+
+**1. Find the base URL for the register(s) you want to use.** 
+
+You can find a full list of register base URLs on our [Registers collection page] (https://registers.cloudapps.digital/registers), but the following are some examples:
```

Comment:
Not sure if `Registers collection page` is the correct term now. Might be worth checking with Connor - https://registers-trial.service.gov.uk/registers

Reply:
I did check with Conor but he wasn't entirely sure going forwards what we'll be calling it, so latest PR just changes it to "here".